improver/calibration/ensemble_calibration.py

```diff
@@ -517,8 +581,8 @@ def calculate_normal_crps(
                 List of optimised coefficients.
                 Order of coefficients is [alpha, beta, gamma, delta].
             forecast_predictor:
-                Data to be used as the predictor,
-                either the ensemble mean or the ensemble realizations.
+                Data to be used as the predictor, either the ensemble mean
```

Comment:
Should this say about the possible static additional forecast predictor

Reply:
I've extended this comment.